README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    - semantic HTML is the goal of writing markup that clearly shows the meaning of each element within the document/webpage.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+    - block level elements will create start a new line on the page and take up the full width available.  Inline elements do not start a new line and only take up as much width as necessary for the content.
```

Comment:
Great definition! many people have trouble understanding the difference but you seem to have a solid understanding of the concept